<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_3" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionNEW_FILE"/>
    <addaction name="actionOPEN_FILE"/>
    <addaction name="actionRECENT_FILES"/>
    <addaction name="actionSAVE_FILE"/>
    <addaction name="actionSAVE_ALL"/>
    <addaction name="separator"/>
    <addaction name="actionCLOSE_FILE"/>
    <addaction name="separator"/>
    <addaction name="actionEXIT"/>
   </widget>
   <widget class="QMenu" name="menuEDIT">
    <property name="title">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHELP">
    <property name="title">
     <string>HELP</string>
    </property>
    <addaction name="actionABOUT"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEDIT"/>
   <addaction name="menuHELP"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNEW_FILE">
   <property name="text">
    <string>NEW_FILE</string>
   </property>
  </action>
  <action name="actionOPEN_FILE">
   <property name="text">
    <string>OPEN_FILE</string>
   </property>
  </action>
  <action name="actionRECENT_FILES">
   <property name="text">
    <string>RECENT_FILES</string>
   </property>
  </action>
  <action name="actionSAVE_FILE">
   <property name="text">
    <string>SAVE_FILE</string>
   </property>
  </action>
  <action name="actionSAVE_ALL">
   <property name="text">
    <string>SAVE_ALL</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="actionCLOSE_FILE">
   <property name="text">
    <string>CLOSE_FILE</string>
   </property>
  </action>
  <action name="actionABOUT">
   <property name="text">
    <string>ABOUT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
